[supervisord]
nodaemon=true
user=root
logfile=/var/log/supervisord.log
pidfile=/var/run/supervisord.pid

[program:cups-management-api]
command=/usr/bin/python3 /usr/bin/cups-management-api.py
directory=/
autostart=true
autorestart=true
startretries=3
stderr_logfile=/var/log/management-api.err.log
stdout_logfile=/var/log/management-api.out.log
user=root
environment=PYTHONUNBUFFERED="1"

[program:cups-ha-integration]
command=/etc/services.d/cups-ha-integration/run
directory=/
autostart=true
autorestart=true
startretries=3
stderr_logfile=/var/log/cups-ha-integration.err.log
stdout_logfile=/var/log/cups-ha-integration.out.log
user=root
environment=PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"

[program:cups-discovery]
command=/etc/services.d/cups-discovery/run
directory=/
autostart=true
autorestart=true
startretries=3
stderr_logfile=/var/log/cups-discovery.err.log
stdout_logfile=/var/log/cups-discovery.out.log
user=root
environment=PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
